src/main/java/org/javacs/InferConfig.java

```diff
@@ -240,12 +257,33 @@ private static String findExecutableOnPath(String name) {
 
     private Set<Path> bazelClasspath() {
         try {
+            // Run bazel as a subprocess
+            String[] output_base_command = {"bazel", "info", "output_base"};
```

Comment:
`output_base` seems to work, but I'm not sure if it is the correct directory. Would it be better to use `execution_base`?

Reply:
oops. I meant `execution_root`.